plugin/chainconfig_manager.go

```diff
@@ -62,3 +62,18 @@ func (c *ChainConfigManager) EnableFeatures(blockHeight int64) error {
 	}
 	return nil
 }
+
+func (c *ChainConfigManager) UpdateConfig() error {
+	settings, err := chainconfig.GetPendingActions(c.ctx, c.build)
+	if err != nil {
+		return err
+	}
+
+	for _, setting := range settings {
+		if err := c.state.SetConfigSetting(setting); err != nil {
+			c.ctx.Logger().Info("failed to apply config change", "key", setting.Name, "err", err)
```

Comment:
`Logger().Error` is probably more approriate

Reply:
Done